src/types.d.ts

```diff
@@ -0,0 +1,25 @@
+module "core/xref" {
```

Comment:
👋👶 "types-escript" noob here ... we have ./src/type-helper.d.ts ... could this go in there?